zebra/zebra_evpn.c

```diff
@@ -1396,9 +1396,10 @@ void zebra_evpn_rem_macip_add(vni_t vni, const struct ethaddr *macaddr,
 	if (ifp)
 		zif = ifp->info;
 	if (!ifp || !if_is_operative(ifp) || !zif || !zif->brslave_info.br_if) {
-		zlog_warn(
-			"Ignoring remote MACIP ADD VNI %u, invalid interface state or info",
-			vni);
+		if (IS_ZEBRA_DEBUG_VXLAN)
```

Comment:
Isn't it better to use IS_ZEBRA_DEBUG_EVPN_MH_MAC here?

Reply:
I think IS_ZEBRA_DEBUG_VXLAN is ok here.